lib/plugins/aws/rollbackFunction/index.js

```diff
@@ -0,0 +1,123 @@
+'use strict';
+
+const BbPromise = require('bluebird');
+const validate = require('../lib/validate');
+const fetch = require('node-fetch');
+
+class AwsRollbackFunction {
+  constructor(serverless, options) {
+    this.serverless = serverless;
+    this.options = options || {};
+    this.provider = this.serverless.getProvider('aws');
+
+    Object.assign(this, validate);
+
+    this.commands = {
+      rollback: {
+        commands: {
+          function: {
+            usage: 'Rollback the function to a specific version',
+            lifecycleEvents: [
+              'rollback',
+            ],
+            options: {
+              function: {
+                usage: 'Name of the function',
+                shortcut: 'f',
+                required: true,
+              },
+              version: {
+                usage: 'Version of the function',
+                shortcut: 'v',
+                required: true,
+              },
+              stage: {
+                usage: 'Stage of the function',
+                shortcut: 's',
+              },
+              region: {
+                usage: 'Region of the function',
+                shortcut: 'r',
+              },
+            },
+          },
+        },
+      },
+    };
+
+    this.hooks = {
+      'rollback:function:rollback': () => BbPromise.bind(this)
+        .then(this.validate)
+        .then(this.getFunctionToBeRestored)
+        .then(this.fetchFunctionCode)
+        .then(this.restoreFunction),
+    };
+  }
+
+  getFunctionToBeRestored() {
+    const funcName = this.options.function;
+    let funcVersion = this.options.version;
+
+    // versions need to be string so that AWS understands it
+    funcVersion = String(this.options.version);
```

Comment:
We should add validation to this option like we do with stage/region and other options. I can run something like `sls rollback function -f hello -v whatever